clients/python-wrapper/tests/integration/test_object.py

```diff
@@ -443,3 +444,17 @@ def test_writer_discard(setup_repo):
         assert writer.closed
 
     assert not obj.exists()
+
+
+@TEST_DATA
+def test_writer_gif(setup_repo, datafiles):
```

Comment:
Neat test!
Can we additionally unit test directly the fileno behaviour? Both are important. For instance if pil start supporting -1 there, but meanwhile other libraries continue to rely on the exception.